src/utils/appDataClient.ts

```diff
@@ -3,12 +3,44 @@ import { getString } from './account'
 
 const client = new Client()
 
-export const fetchHotspots = async () => {
+export const getHotspots = async () => {
   const address = await getString('address')
   if (!address) return []
 
   const newHotspotList = await client.account(address).hotspots.list()
   return newHotspotList.takeJSON(1000)
 }
 
+export const getHotspotDetails = async (address: string) => {
+  const { data } = await client.hotspots.get(address)
+  return data
+}
+
+export const getAccount = async () => {
+  const address = await getString('address')
```

Comment:
I just realized `getString` here reads a bit odd. I know it's not explicitly part of this PR but it took me a minute to figure out what it meant in this context. maybe something like `getSecureValue(key: string)`?

Reply:
yeah, I am totally with you on that.